src/main/java/com/github/dikhan/domain/Incident.java

```diff
@@ -1,127 +1,106 @@
 package com.github.dikhan.domain;
 
-import java.util.List;
 import java.util.Objects;
 
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import javafx.util.Builder;
```

Comment:
commons lang3 from apache has a Builder class, let's use that one instead - it's cleaner as far as the imports go and the package used is more related than fx

import org.apache.commons.lang3.builder.Builder